app/status/healthcheck.py

```diff
@@ -19,8 +19,7 @@ def show_status():
     else:
         return jsonify(
             status="ok",  # This should be considered part of the public API
-            travis_commit=version.__travis_commit__,
-            travis_build_number=version.__travis_job_number__,
+            github_commit=version.__git_commit__,
```

Comment:
`git` is used in some places and `github` in others, so it would be nice if that was consistent across the files. I prefer `git`, since GitHub is just the tool to view the git commits.